components/console-backend-service/internal/domain/k8s/namespace_resolver.go

```diff
@@ -95,3 +104,18 @@ func (r *namespaceResolver) ApplicationsField(ctx context.Context, obj *gqlschem
 
 	return appNames, nil
 }
+
+func (r *namespaceResolver) CreateNamespaceMutation(ctx context.Context, name string, labels gqlschema.Labels) (gqlschema.NamespaceCreationOutput, error) { //namespaceoutput
+
+	// temporary: add 'env' label to newly created namespaces
```

Comment:
Instead of describing the code (which is pretty obvious) I would explain why it is temporary and when it can be removed